LandSandBoat/server
# Fix overlapping indexes in Cloister of Frost Battlefields

**_I affirm:_**

## What does this pull request do?
Corrects an overlapping index in Cloister of Frost BCNMs that will cause players intending to enter the Trial by Ice BCNM to land in Class Reunion instead.

## Steps to test these changes
!togglegm and Enter each battlefield, see that you will land in the correct one.
